server/tests/integration/automation-scheduler.test.js

```diff
@@ -8,6 +8,7 @@ const { getAtVersions } = require('../../models/services/AtService');
 const { getBrowserVersions } = require('../../models/services/BrowserService');
 const dbCleaner = require('../util/db-cleaner');
 const { default: axios } = require('axios');
+const { COLLECTION_JOB_STATUS } = require('../../util/enums');
```

Comment:
I am a little on the fence. My thinking is that reduced friction in editing and a single source of truth would ensure that any changes to the external systems could be quickly matched and verified on the app. The original motivation for this was that I had a typo with a status of "CANCELED" and it wasn't being caught because semantic satiation + copy/paste was causing me to repeat the mispelling elsewhere. It feels safer to me to have a single source of truth that ensures consistent breakdown in all uses if deviation ever with external system occurs. Not a strongly held opinion, so would be happy to change.

Reply:
Let's keep it as-is, then!

(Since I get the sense that this kind of thing interests you, too: we agree on the value of a single source of truth, it might just be that my definition is contained to the application logic. My concern taken ad absurdum might look like a test suite that simply verifies that internal functions are being called.)

Reply:
Ah! Yes, now I better understand your comment and am in complete agreement with you. I took your comment to be on the existence of the enum largely but now I have re-read your first sentence closely and see the word test. I agree with using literal values in the tests. That is useful friction. Changed it 😆

Reply:
Oh, cool! Thanks for continuing to consider this so thoughtfully